lib/libsys/intro.2

```diff
@@ -261,11 +260,7 @@ or some other network limit.
 .It Er 41 EPROTOTYPE Em "Protocol wrong type for socket" .
 A protocol was specified that does not support the semantics of the
 socket type requested.
-For example, you cannot use the
-.Tn ARPA
-Internet
-.Tn UDP
-protocol with type
+For example, you cannot use the ARPA Internet UDP protocol with type
```

Comment:
"ARPA Internet" seems redundant. I suppose this text dates to a time when OSI seemed potentially relevant?